operators/pkg/forge/containers.go

```diff
@@ -470,7 +476,13 @@ func ContainerVolume(volumeName, claimName string, environment *clv1alpha2.Envir
 // NeedsContainerVolume returns true in the cases in which a volume mount could be needed.
 func NeedsContainerVolume(instance *clv1alpha2.Instance, environment *clv1alpha2.Environment) bool {
```

Comment:
I was wondering whether it wouldn't be better to always create the container volume, for the sake of simplicity (thus dropping this function). After all, it seems there would be practically no real case in which it is not created.

Reply:
Followed your suggestion and removed this method, always adding the volume